app/jobs/participant_outcomes/stream_api_requests_to_big_query_job.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ParticipantOutcomes
+  class StreamApiRequestsToBigQueryJob < ApplicationJob
+    queue_as :big_query
+
+    BQ_TABLE_NAME = "npq_participant_outcome_api_requests"
+
+    def perform(participant_outcome_api_request_id:)
+      api_request = ParticipantOutcomeApiRequest.find(participant_outcome_api_request_id)
+
+      rows = [
+        {
+          participant_outcome_api_request_id: api_request.id,
+          request_path: api_request.request_path,
+          status_code: api_request.status_code,
+          request_headers: api_request.request_headers,
```

Comment:
discussed in person: all `jsonb` fields need to have `to_json` appended to work in Biq query :)

Reply:
updated 👍

Reply:
could you also add those to the headers fields as well please?

Reply:
done 👍